08.Lesson_WorkingWithInvariants/README.md

```diff
@@ -17,6 +17,7 @@ Upload your solutions to the respective directories for a review by the Certora
 ## Vacuous Rules
 
 </br>
+- [ ] look at the solutions folder to check your spec
```

Comment:
Wrong representation of this line in view mode. Maybe need an empty line between </br> and text line?